content/components/core/code/scss-variables.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: section
+---
+
+## SCSS Variables
+
+### Default Theme Colours
+
+| Variable Name               | Default Value | Description                                                                                                |
+|-----------------------------|---------------|------------------------------------------------------------------------------------------------------------|
+| `$AU-color-foreground-text`   | `#313131`       | Colour used for paragraphs, headings etc.                                                                                 |
+| `$AU-color-foreground-action` | `#00698F`       | Colour of links, buttons or other action items.                                                             |
+| `$AU-color-foreground-focus`  | `#9263DE`       | Color around an element that is in focus state. |
```

Comment:
Around is kind of right, we can use `outline` though as it is simple and technically correct. `Colour that outlines an element when it receives focus`